### PR TITLE
chore: update docker for 1.0.0a5

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: docker/pyproject.toml [hunk 1]
```diff
@@ -22,4 +23,4 @@ isort = "^5.9.1"
 pylint = "^2.8.3"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
\ No newline at end of file
+build-backend = "poetry.core.masonry.api"
```
Comment (parano) [inline docker/pyproject.toml:26]: cc @bojiang - `./docker` is a separate python project, make sure to update its dependency when making changes

File: docker/pyproject.toml [hunk 2]
```diff
@@ -14,6 +14,7 @@ Cerberus = "^1.3.4"
 glom = "^20.11.0"
 python-dotenv = "^0.18.0"
 GitPython = "^3.1.18"
+"backports.cached-property" = "^1.0.1"
```
Comment (parano) [inline docker/pyproject.toml:17]: cc @bojiang - `./docker` is a separate python project, make sure to update its dependency when making changes

**Comment by parano** [comment]
@aarnphm how do we update the poetry.lock file here?

File: docker/manifest.yml [hunk 3]
```diff
@@ -75,23 +75,24 @@ releases:
...
 
-  centos8:
-    <<: *base_spec
-    templates_dir: templates/rhel
-    base_image: centos:8
-    add_to_tags: "centos8"
-    cuda:
-      <<: *cuda11_3_1
-    cuda_prefix_url: "rhel8"
+  # TODO: migrate to centos stream
```
Comment (parano) [inline docker/manifest.yml:78]: skipping centos base image release for 1.0.0a5 for now, due to CentOS 8 EOL. will discuss further with the team

**Comment by aarnphm** [comment]
> @aarnphm how do we update the poetry.lock file here?

you just do `poetry update`

File: docker/manifest.yml (hunk 3 above)
Comment (aarnphm) [reply docker/manifest.yml:78]: I will address ubi in a separate issues

**Review by aarnphm — APPROVED** [review]

**Pull request merged**